src/Forwarder.cpp

```diff
@@ -176,7 +174,14 @@ void Forwarder::forward_epics_to_kafka() {
 
   using namespace std::chrono_literals;
   std::atomic<MILLISECONDS> IterationExecutionDuration(0ms);
```

Comment:
```suggestion
```
While we are here anyway. Please remove this abomination.

Reply:
Forwarder main loop is being cleaned up as part of DM-1755